src/vm/errors/vm_errors.rs

```diff
@@ -120,6 +122,8 @@ impl fmt::Display for VirtualMachineError {
             },
             VirtualMachineError::UnknownHint(hint_code) => write!(f, "Unknown Hint: {:?}", hint_code),
             VirtualMachineError::MemoryError(memory_error) => memory_error.fmt(f),
+            VirtualMachineError::SplitIntNotZero => write!(f,"split_int(): value is out of range"),
```

Comment:
Why not include the value?

Reply:
The value that is out of range is the one that the function split_int (and therefore this hint) receives when it its first called, this case is reached after n recursive calls, where the value received by the hint is no longer the one received by the function, so it doesn't make sense to show this number to the user.